### DISCUSSION TITLE
how to get path to config yaml?

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [Platonic-Systems/process-compose-flake#48] [Q&A]
I have it here `/nix/store/r8342phg32394gppfmmqxwdynhh4chj3-devnet-xc.yaml`

I want to run `process-compose down /nix/store/r8342phg32394gppfmmqxwdynhh4chj3-devnet-xc.yaml`

But seems config is hidden inside package, so I in theory can find deps of package, and find yaml and that yaml would be the thing. So need to parse package stucture for this.

**Comment by srid** [comment]
Can't you pass `down` to the argument of resulting package? Your CLI arguments are passed as is to process-compose: 

https://github.com/Platonic-Systems/process-compose-flake/blob/fc19f300807fccc2543a3a4c8c9beb66ce96c7a4/nix/process-compose/default.nix#L52C19-L52C19

---

The config file paths are available in the `PC_CONFIG_FILES` environment variable (to the processes), as well as in the `--help` text output (if invoking from the CLI). 

```
❯ nix run ./example -- --help | grep PC_CONFIG_FILES
  -f, --config stringArray   path to config files to load (env: PC_CONFIG_FILES) (default [/nix/store/8j1rnipbcmcyqg6md3mdizx6i4qwagdl-default.yaml])
```

**Reply by srid** [reply]
Also note that the `down` command was added a month ago,
https://github.com/F1bonacc1/process-compose/commit/fbb29bd4eae25ace491210c052c2eacd5411bffc

So your nixpkgs process-compose may not have it.
